src/backgroundScript.js

```diff
@@ -0,0 +1,42 @@
+export const backgroundScript = (selectors, colors) => {
```

Comment:
How about a default export? can we do that? We are not exporting anything more than a backgroundScript so that could be a improvement?

Reply:
I thought about it, probably does not matter much, what do you prefer or what is the better practice?